packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -348,6 +358,38 @@ module.exports = {
     // https://github.com/jmblog/how-to-optimize-momentjs-with-webpack
     // You can remove this if you don't use Moment.js:
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+
+    // For some reason, Webpack adds some ids of all the modules that exist to our vendor chunk
+    // Instead of using numerical ids it uses a unique path to map our request to a module.
+    // Thanks to this change the vendor hash will now always stay the same
+    new webpack.NamedModulesPlugin(),
+
+    // Ensure that every chunks have an actual name and not an id
+    // If the chunk has a name, this name is used
+    // otherwise the name of the file is used
+    new webpack.NamedChunksPlugin(chunk => {
+      if (chunk.name) {
+        return chunk.name;
+      }
+      return (chunk.mapModules
+        ? chunk.mapModules(m => m.request)
+        : chunk.modules.map(m => m.request)
```

Comment:
I don't think we need to have fallback behavior for old webpack versions, or is this here for another reason?

Reply:
mapModules isn't supported in Webpack 2.6 and modules are deprecated in Webpack 3

Reply:
We don't need to worry about webpack 2.6 support 😄

Reply:
It was the current version on CRA when I fork the repo
But now the v3 is used 😁

Reply:
I'll pull the current version of CRA